packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -122,3 +122,19 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
     target: pipeTarget
 });
 ```
+
+### Amazon CloudWatch Logs Log Group
```

Comment:
I understand that CloudWatch Logs is the product name, but it sounds pretty redundant here
```suggestion
### Amazon CloudWatch Log Group
```

Reply:
I guess there's already some precedence:
<img width="520" alt="image" src="https://github.com/aws/aws-cdk/assets/2505696/646e47d9-8f85-49c7-b21c-66aba32feb39">

Logs is missing more often than not, but I think that the product name change from "CloudWatch" "CloudWatch Logs" is somewhat recent?

<img width="521" alt="image" src="https://github.com/aws/aws-cdk/assets/2505696/8c13da03-abab-4459-bffa-2833f163ce62">

I'm not dead set on changing it, I'll leave it up to you

Reply:
Definitely redundant but the service is `Amazon CloudWatch Logs` and the product is a `log group`, usually lowercase but since it's in a title I capitalized it here. I'll leave this as-is.